pkg/kritis/review/validating_transport.go

```diff
@@ -41,7 +43,7 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 	keys := map[string]string{}
 	key, fingerprint, err := secrets.KeyAndFingerprint(avt.Attestor.Spec.PublicKeyData)
 	if err != nil {
-		glog.Errorf("Error parsing key for %q: %v", avt.Attestor.Name, err)
+		glog.Infof("Error parsing key for %q: %v", avt.Attestor.Name, err)
```

Comment:
What was the rationale behind downgrading the severity of the log message here?

Reply:
I don't know :)  Restoring to the way it was.